packages/server/src/internals/server/index.ts

```diff
@@ -41,7 +41,10 @@ export async function start() {
   })
   initializeSentry({ context: 'server' })
   const timer = createTimer()
-  const cache = createCache({ timer })
+  const cache =
+    process.env?.CACHE_TYPE !== 'production'
```

Comment:
```suggestion
    process.env.CACHE_TYPE === 'empty'
```
That way we don't need to add a new env var in production, just in local development.
Besides that it would be good to add the new env var in .env and in packages/server/src/env.d.ts
Besides that, it is good to have some kind of